src/main/java/binary_search/OccuranceOfElement.java

```diff
@@ -0,0 +1,37 @@
+package binary_search;
+    public class OccuranceOfElement {
+        public int countOfElement(int[] nums,int target){
```

Comment:
method name can be improved to:
occuranceOfElement
numberOfOccurance
frequencyCount